comps/comps-foreman-rhel7.xml

```diff
@@ -42,7 +41,26 @@
 
       <packagereq type="default">tfm-mod_passenger</packagereq>
 
+      <!-- Assets dependencies -->
```

Comment:
I started doing this to add `nodejs-jquery` here, then it's when I realized it'd be only non-scl so it had to be in other section. Anyway I kept these sections just in case.